docs/en/ingest-management/data-streams.asciidoc

```diff
@@ -268,6 +268,18 @@ PUT _ingest/pipeline/metrics-system.cpu@custom
 Custom pipelines can directly contain processors or you can use the pipeline processor to call other pipelines that can be shared across multiple data streams or integrations.
 These pipelines will persist across all version upgrades.
 
+[[data-streams-pipelines-warning]]
+[WARNING]
+====
+If you have a custom pipeline defined that matches the naming scheme used for any {fleet} custom ingest pipelines, this can produce unintended results. For example, if you have a pipeline named like one of the following:
+
+* `global@custom`
+* `traces@custom`
+* `traces-apm@custom`
```

Comment:
Thanks for including some examples. I'll work on getting a full set of collision candidates (if it even exceeds this) on https://github.com/elastic/kibana/issues/175254